operator/src/main/java/oracle/kubernetes/operator/DomainStatusUpdater.java

```diff
@@ -611,6 +611,16 @@ void modifyStatus(DomainStatus status) {
     }
   }
 
+  /**
+   * A utility method for unit testing.
+   */
+  public static Step excludeProgressingStep(Step step) {
```

Comment:
I understand that the ProgressingStep is a private static class, but I think it is better to have this method to unit test code.

Reply:
I initially had this method in the test code, and made the ProcessingStep public. I changed to the current implementation because I slightly preferred keeping the class private. But I am okay with either approach.